rspace/src/test/scala/coop/rchain/rspace/ReplayRSpaceTests.scala

```diff
@@ -745,6 +745,42 @@ class ReplayRSpaceTests extends ReplayRSpaceTestsBase[String, Pattern, String, S
 
       replaySpace.produce(channel, datum, persist = false) shouldBe defined
   }
+
+  "clear" should "empty the store, reset the event log, reset the trie updates log, and reset the replay data" in
```

Comment:
huh I might be misreading, but shouldn't the description be: "empty the replayStore, reset the replay event log, reset the replay trie updates log, and reset the replay data"? or some combination of the above? the rspace store isn't tested AFAI can see.

Reply:
yes, this description does suffer from a bit of copy-paste.  will do.